### DISCUSSION TITLE
Do you want your bot featured as a production sample?

### DISCUSSION THREAD

**Discussion by csmir** [discord-net/Discord.Net#2114] [Show and tell]
### 📚 New sample system

In the near future, a new sample system will be introduced to allow new Discord.Net users to more easily find, traverse and learn from samples. Right now, the samples get little love, and while functional as a base, also don't really show much of what the actual capabilities of this library are.

### 🤖 Getting your bot featured

There are a few requirements to get your bot featured as a sample:

- **Your repository needs to be public.** 
   - If made private after being featured, your sample will be removed as it is no longer accessible.
- **Your repository needs to be under MIT licensing.** 
  - The code will be accessible to developers who may want to use parts of it in production, without having to come to agreement with the license holder.
- **Your code will need to be readable, and documented.**
  - There is no point in having samples that are undocumented or written without respect to conventions.

**Comment by bman46** [comment]
If you would like to use my Instagram Embed bot as a sample, you are more than welcome too.
https://github.com/bman46/InstagramEmbedDiscordBot
